src/zenml/new/pipelines/code_archive.py

```diff
@@ -1,157 +0,0 @@
-#  Copyright (c) ZenML GmbH 2024. All Rights Reserved.
```

Comment:
Just moved the files, but I think the reason the diff is so weird is because there were old files with the same name but different content (e.g. the old pipeline/step decorator that behaved differently)